### PR TITLE
Address linking fix

### PR DESCRIPTION
**Pull request by dillchen**
## Description

Fixes an import issue with ethUtil which was preventing a new Eth address from being able to be linked.

## Motivation and Context

Getting ETH and Moloch working.

## How Has This Been Tested?

Log in using ETH wallet.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] My change should be included in the release notes.
- [ ] I have updated the documentation accordingly.
- [ ] I have linted the code locally prior to submission.

### CODE REVIEW

File: client/scripts/controllers/chain/ethereum/main.ts [hunk 1]
```diff
@@ -38,10 +39,14 @@ class Ethereum extends IChainAdapter<EthereumCoin, EthereumAccount> {
     await this.accounts.init(this.chain);
     await this.chain.initEventLoop();
 
-    await this.webWallet.web3.givenProvider.on('accountsChanged', function (accounts) {
-      const updatedAddress = this.app.login.activeAddresses.find((addr) => addr.address === accounts[0])
-      selectLogin(updatedAddress);
-    });
+    if (this.webWallet) {
+      const app = this.app;
```
Comment (drewstone) [inline client/scripts/controllers/chain/ethereum/main.ts:43]: why add this when you can call it directly below? `this.app.login.activeAddresses`

File: client/scripts/controllers/chain/ethereum/moloch/adapter.ts [hunk 2]
```diff
@@ -54,7 +54,17 @@ export default class Moloch extends IChainAdapter<MolochShares, EthereumAccount>
     const api = new MolochAPI(this.meta.address, this.chain.api.currentProvider, activeAddress);
     await api.init();
 
+    if (this.webWallet) {
+      const app = this.app;
```
Comment (drewstone) [inline client/scripts/controllers/chain/ethereum/moloch/adapter.ts:58]: same as above, this isn't needed.

File: package.json [hunk 3]
```diff
@@ -12,7 +12,6 @@
     "lint": "./scripts/lint-new-work.sh",
     "lint-all": "eslint client/\\**/*.ts server/\\**/*.ts",
     "test-client": "webpack-dev-server --config webpack/webpack.config.test.js",
-    "unit-test": "nyc ts-mocha --project tsconfig.node.json ./test/unit/**/*.spec.ts",
```
Comment (drewstone) [inline package.json:15]: thanks!

File: client/scripts/controllers/chain/ethereum/main.ts [hunk 4]
```diff
@@ -3,8 +3,9 @@ import { default as EthereumAccounts, EthereumAccount } from 'controllers/chain/
 import { EthereumCoin } from 'shared/adapters/chain/ethereum/types';
 import { IChainAdapter, ChainBase, ChainClass } from 'models';
 
-import EthWebWalletController from '../../app/eth_web_wallet';
-import { selectLogin } from '../../app/login';
+import EthWebWalletController from 'controllers/app/eth_web_wallet';
+import { selectLogin } from 'controllers/app/login';
+import { toChecksumAddress } from 'web3-utils';
```
Comment (drewstone) [inline client/scripts/controllers/chain/ethereum/main.ts:8]: doesn't appear to be used?

**Comment by raykyri** [comment]
@drewstone The `const app = this.app` assignments were because we were using function() declarations instead of arrow functions. We should be using arrow functions anyway so I have fixed that.

**Pull request merged**